src/components/Books/Address.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Address = props => (
+  <div className="item-address">
+    <span>{props.distance}</span>
+    <span className="separator">{props.separator}</span>
+    <span>{props.name}</span>
+  </div>
+);
+
+Address.propTypes = {
+  distance: PropTypes.string,
+  separator: PropTypes.object,
+  name: PropTypes.string
+};
+
+export default Address;
```

Comment:
What is this component for? I didn't see any other components calling this component

Reply:
@Envl said that he will use this later.